cypress/e2e/shared/fluxQueryBuilder.test.ts

```diff
@@ -303,7 +301,6 @@ describe('Script Builder', () => {
       loginWithFlags({
         schemaComposition: false,
         newDataExplorer: true,
-        saveAsScript: false,
```

Comment:
Seems like the test fails at line 309 because there is no longer a "Clear" button